src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/index.tsx

```diff
@@ -90,6 +103,19 @@ const LoadingDimmer = styled.div<{ visible: boolean }>`
   visibility: ${(props) => (props.visible ? 'visible' : 'hidden')};
 `
 
+const useWindowSize = () => {
```

Comment:
This hook is used to get the windowSize which is used for android.
On android when you focus on a textfield the keyboard pushes everything up (resizes the window). 
The reason for not using focus/blur is that, if you close the keyboard on android the input element still has focus.
When the keyboard is up (element is focused) the "update details button" needs to be positioned relative to not look wonky.
**(OBS)
This behaviour should not be the same on Iphone and I cant test this particular case using a simulator.**